resources/assets/js/app.js

```diff
@@ -173,7 +173,10 @@ $(document).ready(function () {
                     continue;
                 }
             }
-            data.append(this.elements[i].name, this.elements[i].value)
+            // Add checkboxes only if they are checked
+            if(e.currentTarget.elements[i].type != 'checkbox' || e.currentTarget.elements[i].checked) {
```

Comment:
I think I prefer `== &&` as opposed to `!= ||` as you have currently.

Reply:
I commited the desired changes.